modape/modis/download.py

```diff
@@ -234,55 +270,72 @@ def download(self,
             overwrite (bool): Replace existing files.
             multithread (bool): Use multiple threads for downloading.
             nthreads (int): Number of threads.
+            max_retries (int): Maximum number of retries for failed downloads (for no max, set -1).
+            robust (bool): Perform robust downloading (checks file size and checksum).
 
         Raises:
             DownloadError: If one or more errors were encountered during downloading.
+        Returns:
+            List of downloaded MODIS HDF file IDs.
```

Comment:
Effectively: file names

Reply:
Well yes, but not the full path

Reply:
Exactly. So, IDs that are "meaningful" in the user's local system. Namely: filenames in the targeted download directory. Just imagine the "MODIS HDF file IDs" were numbers in some remote system... Pretty useless. I think it is really valuable to point out at this point of documentation that a plain and easy list of filenames is returned